### DISCUSSION TITLE
Is there way to resolve import .proto file problem

### DISCUSSION THREAD

**Discussion by shoppingjaws** [stepci/stepci#127] [Q&A] [answered]
Hi, this is great product.🙌 I'm very impressed.
As the title says, stepci doesn't solve import path on gRPC.
files are below.

- ./proto/api.proto
api.proto imports annotations.proto
- workflow.yaml
- ./../googleapis/annotations.proto

Is there way to let stepci known the import path of annotations.proto

Best

**Comment by mishushakov** [comment, marked as answer]
Hey,
thanks for your compliment!

We're working on it! I will let you know once we have something

**Comment by sebastianwis** [comment]
Hey @nakamuloud 
Thank you for your contribution here! 
It would be amazing to welcome you to our Discord if you haven't joined it yet: 
https://discord.com/invite/KqJJzJ3BTu
This is where we keep users up to date with updates, features and bug resolutions.

**Comment by mishushakov** [comment]
Hey @nakamuloud, we just improved our gRPC integration to work with proto-loader in runner version 1.11.0
